api/src/main/kotlin/nl/avisi/anowalke/services/translation/DummyVertaalService.kt

```diff
@@ -0,0 +1,17 @@
+package nl.avisi.anowalke.services.translation
+
+import nl.avisi.anowalke.TranslateConnector
+import nl.avisi.anowalke.dto.ExpressionsDto
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Service
+
+@Service
+@ConditionalOnProperty(name = ["translation.service"], havingValue = "dummy")
+class DummyVertaalService: TranslateConnector {
+    override val api_key = null
```

Comment:
Zo te zien dus niet generiek en enkel voor Microsoft implementatie relevant?

Reply:
Dat ligt eraan. Een dummy is natuurlijk een apart verhaal maar elke andere service zal het wel nodig hebben.